### PR TITLE
Add LSSTComCamSim as allowed camera type.

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ jobs:
 
       - name: Install
         run: |
+          $CONDA/bin/conda install python=3.11 -y
```
Comment (suberlak) [inline .github/workflows/lint.yaml:20]: if you rebase against develop I think this won't be needed

**Review by suberlak — APPROVED** [review]
Do we have any files that would have that instrument name in the header to test that they can be processed ?

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (jbkalmbach) [reply .github/workflows/lint.yaml:20]: I'll keep`-y` in though to avoid any issues with needing confirmation in github actions.

**Comment by jbkalmbach** [comment]
> Do we have any files that would have that instrument name in the header to test that they can be processed ?

Demonstration that the closed loop creates `LSSTComCamSim` headers and they are ingested in the butler repository as such is found in the JIRA issue: https://rubinobs.atlassian.net/browse/DM-43221

Also shows that the closed loop successfully converges with `LSSTComCamSim` images.

**Pull request merged**